easybuild/easyconfigs/h/htop/htop-2.0.1.eb

```diff
@@ -0,0 +1,29 @@
+easyblock = 'ConfigureMake'
+
+name = 'htop'
+version = "2.0.1"
+
+homepage = 'http://hisham.hm/htop/'
+
+description = """An interactive process viewer for Unix"""
+
+toolchain = {'name': 'dummy', 'version': ''}
+
+source_urls = ['http://hisham.hm/htop/releases/%(version)s/']
+sources = [SOURCE_TAR_GZ]
+checksums = ['f410626dfaf6b70fdf73cd7bb33cae768869707028d847fed94a978e974f5666']
+
+dependencies = [
+    ('ncurses', '6.0'),
+]
+
+prebuildopts = """
+    sed -e 's/^LIBS = -lncursesw -lm/LIBS = -lncursesw -lm -ltinfo/' -i.eb Makefile
```

Comment:
I don't think we need to patch the `Makefile` at runtime like this here, this looks equivalent with:

```python
buildopts = 'LIBS="-lncursesw -lm -ltinfo"'
```

(assuming that the original value for `LIBS` doesn't have anything else than `-lncursesw -lm`